caddyhttp/fastcgi/fastcgi.go

```diff
@@ -465,3 +480,11 @@ type LogError string
 func (l LogError) Error() string {
 	return string(l)
 }
+
+// Map of supported protocols to Apache ssl_mod format
+// Note that these are slightly different from SupportedProtocols in caddytls/config.go's
+var sslProtocolToStringMap = map[uint16]string{
```

Comment:
Rename to `tlsProtocolStringToMap`

Also, was the second sentence unfinished? Include punctuation; to clarify. :)

Reply:
Is there a preference for how these things should be named? I noticed some inconsistency in the caddy source and just went with something that made sense (to me, but YMMV).

Reply:
It should be `tlsProtocolStringToMap` (to be consistent)

Reply:
Okay, two maps it is; but yes rename to `tls...`